main.go

```diff
@@ -47,7 +52,11 @@ func main() {
 
 	notifiers := make(map[string]chan notifier.Message)
 	go notifier.SetupStdErrNotifier(notifiers)
-	go notifier.SetupUnixSocketNotifier(notifiers, exits)
+	if libnotify {
+		go notifier.SetupLibnotifyNotifier(notifiers, libnotifyDuration)
+	} else {
+		go notifier.SetupUnixSocketNotifier(notifiers, exits)
```

Comment:
Let's not do exclusive `if-else` and keep unix sockets enabled at all times, not to break integration with other apps. For example user might want to let `py3status` still use sockets to show an icon in status bar and _also_ to get notifications via libnotify.

Reply:
Ok.